ssc/cmod_battery.h

```diff
@@ -27,6 +27,8 @@ OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "core.h"
 #include "lib_battery.h"
+#include "lib_utility_rate.h"
```

Comment:
lib_utility_rate holds forecasting classes used by the various battery modules. I would think any changes would be more likely to show up in lib_utility_rate_equations. That said, I'm open to naming changes if the names are confusing.

Reply:
See https://github.com/NREL/SAM/issues/361#issuecomment-691233665 for a summary. Most of that seems like it would be handled in the JSON processing/setup (specifying min charge and its units rather than having annual and monthly fields) than in downstream calculation code.